nemo/utils/exp_manager.py

```diff
@@ -872,6 +879,20 @@ def configure_loggers(
         logger_list.append(clearml_logger)
         logging.info("ClearMLLogger has been set up")
 
+    if create_neptune_logger:
+        if neptune_kwargs is None:
+            neptune_kwargs = {}
+        if "name" not in neptune_kwargs and "project" not in neptune_kwargs:
+            raise ValueError("name and project are required for neptune_logger")
```

Comment:
Check at import section at the top of Neptune is installed with try catch pattern, and raise here if it's not available.

Reply:
This is already being handled by PyTorch lightning inherently to fail with proper messages. Would we still require it here?
Ref [this](https://github.com/Lightning-AI/pytorch-lightning/blob/d02009af76531e803a6e1593827894b436e366fc/src/lightning/pytorch/loggers/neptune.py#L226) line in PyTorch Lightning repository.

If its still required here, I am more than happy to duplicate the same checks here. Had considered it before but opted against considering code bloat.